src/components/Form/Form.jsx

```diff
@@ -15,12 +16,37 @@ export const Form = () => {
     lastDate: null,
   });
 
+  const [value] = useCollection(getAll());
+
+  const isItemDuplicated = (name) => {
+    const regex = /[!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~]/g;
+
+    if (value) {
+      const item = value.docs.filter(
+        (doc) =>
+          doc.data().name.toLowerCase().replace(regex, '') ===
+          name.toLowerCase().replace(regex, ''),
+      );
+      return item.length ? true : false;
```

Comment:
Here's a fun trick. Usually if you find yourself writing a ternary that evaluates to `? true : false`, you can just eliminate the ternary entirely and return the conditional part of it to the left of the `?` with a `!!`:-)

In other words. `item.length ? true : false` is logically equivalent to `!!item.length`